.github/workflows/api-waf-dpc-apply.yml

```
@@ -0,0 +1,35 @@
+name: api-waf plan terraform
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/api-waf-plan.yml
+      - terraform/services/api-waf/**
+  workflow_dispatch: # Allow manual trigger
+
+jobs:
+  terraform-apply:
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./terraform/services/api-waf
+    strategy:
+      fail-fast: false
+      matrix:
+        app: [dpc]
+        env: [dev, test, sbx, prod]
```

Let's start with dev environment only

```suggestion
        env: [dev]
```

---

Applied